.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: weekly
+    ignore:
+        # cannot be updated until the etcd library is updated
+      - dependency-name: "google.golang.org/grpc"
+        # using a cilium-specific fork
+      - dependency-name: "github.com/miekg/dns"
```

Comment:
I think this isn't necessary either. `github.com/miekg/dns` is not a direct Hubble dependency.

Reply:
I left this in because the `github.com/miekg/dns` override is explicitly in `go.mod`, but I'll try removing it.